### ISSUE TITLE
Expletive there (proposal)

### ISSUE THREAD

**Issue by omriabnd** [UniversalConceptualCognitiveAnnotation/docs#53]
Expletives are always Fs.

Where there is no relation word present, we add an implicit unit:
"There_F are_F [thousands_Q [of_R elephants_C]_C]_A [where we live]_A"
"There_F are_F [two boys and five girls]_A"

Where there is a relation word present, be it a preposition, a verb or some other word, we mark it as the main relation:
"There_F are_F [lots_Q [of_R them_C]_C]_A lurking_P [in_R the_F bushes_C]_A"
"There_F are_F earrings_A on_S [the_F table_C]_A"

Where there are multiple relations, each of them is an H:
[There_F is_F a_F new_S building_A]_H [(building)_A on_S Campus_A]_H [(building)_A with_S presumably_G [better_E facilities_C]_A ]_E]_A
[There_F is a lecture_P]_H [I recommend [you hear (lecture)_A]_A]_H

**Comment by nschneid** [comment]
What do you propose for:

There is a restaurant nearby.
Nearby, there is a restaurant.
There is a nearby restaurant.
A restaurant is nearby.

If I'm not mistaken, the existential construction licenses 2 arguments: a thing that exists, and optionally, its location. I parse "There is a new building on campus with better facilities" as a nonprojective version of "There is [a new building with better facilities] [on campus]". Not sure if that's relevant.

Also, I'm not sure why you don't consider "where" to be a "relation word".

**Comment by omriabnd** [comment]
**Nathan's examples:**
All of them have a single relation ("nearby") and hence a single scene:
There_F is_F [a restaurant]_A nearby_S
Nearby_S, there_F is_F [a restaurant]_A
There_F is_F a_F nearby_S restaurant_A
[A restaurant]_A is_F nearby_S.
[A restaurant]_A is_F located_S nearby_D.

Re licensing of arguments: I'm not sure that we want to go according to the syntactic licensing, because then "with better facilities" is not on the same status as "on campus" and so the following two examples get different treatment: (though we do want to treat them the same)
1. There is a new building on campus with better facilities
2. There is a new building with better facilities on campus

Corrections for before: 

Where there is no relation word present, we add an implicit unit:
"[There_F are_F [thousands_Q [of_R elephants_C]_C]_A (IMPLICIT)_S] where_L [we live]_A"
"There_F are_F [two boys and five girls]_A (IMPLICIT)_S"

Where there is a relation word present, be it a preposition, a verb or some other word, we mark it as the main relation:
"There_F are_F [lots_Q [of_R them_C]_C]_A lurking_P [in_R the_F bushes_C]_A"
"There_F are_F earrings_A on_S [the_F table_C]_A"

Where there are multiple relations, each of them is an H:
[There_F is_F a_F new_S building_A]_H [(building)_A on_S Campus_A]_H [(building)_A with_S presumably_G [better_E facilities_C]_A ]_E]_A
[There_F is a lecture_P]_H [I recommend [you hear (lecture)_A]_A]_H

**Comment by nschneid** [comment]
I worry that the designation of which modifier is scene-evoking is subjective and potentially a slippery slope.

* Nearby, there is a new neighborhood restaurant with excellent food.
  - "nearby", "new", "neighborhood", and "with excellent food" are all modifiers of the restaurant. Are they all evokers of top-level scenes?

Recall our discussions of sentences like "This is a great restaurant". There was temptation to make "great" the main scene-evoker, but this is a bit subjective (is the main point that it is a restaurant, or that it is a great one?) and especially problematic when there are multiple modifiers.

Consider also something like 

* There is nobody in the room who knows the answer.
  - I would say the most salient piece of information is the negation in "nobody", and "in the room" just qualifies which set of people it is in reference to; it is not about explaining where nobody is.

**Comment by omriabnd** [comment]
Nathan's prposal: where existence and location are both expressed,
existence is secondary to location.

**Comment by omriabnd** [comment]
So here are the new guidelines for "there". 

There is a hierarchy of semantic relations that could appear in a "there" construction:
(1) Existential
(2) Location / other prepositional phrase that normally would not evoke a Scene
(3) Scene-evoking unit

Where only (1) appears in the sentence, "there" is the S.
Where (1), (2) appear in the sentence, the location will be the S. Example: "there_F are_F earrings_A on_S [the table]_A.", "there_F is_F a_F new_S [building_C [with a tilted roof]_E]_A"
Where (3) appears in the sentence (with or without (2)), it is the S. If (2) appears as well, it does not evoke a  separate Scene. Examples: "there_F is_F a_F great_S restaurant_A nearby_A", "there_F is_F a_F new_S building_A [on campus]_A [with new facilities]_A".

Where there are several relations of type (3), they evoke separate scenes: "[there_F is_F a_F new_S]_H- [[big_S] (MANY REMOTES HERE)]_H [restaurant_A [nearby]_A [with better facilities]_A]_-H" .

**Comment by omriabnd** [comment]
We can also add "syntactic sugar": coordinated states, which in post-processing is separated to different Hs.

**Comment by nschneid** [comment]
Why not there_D instead of there_F?

There_F suggests to me that it is likely not to be contributing content, and thus could be omitted in other languages. But I think the existential information in "there are earrings on the table" is contentful, even if not the main relation.

**Comment by omriabnd** [comment]
I agree with you in principle but as the guidelines state that "there"
should be an F in these cases, and since this has already been entrenched,
I'd defer this to version 2.1.

**Comment by nschneid** [comment]
Rediscussed via email in Nov 2020 in preparing the UCCA tutorial at COLING. Conclusion for v2.1 guidelines:

> How about we just say that existential "There" is S unless (a) there is a locative phrase after the entity, or (b) the thing that exists is scene-evoking?
>
> There_S is [a restaurant]_A.                 - cf. guidelines "There are thousands of us"
> There_S is [a great restaurant]_A.
> There_F is [a restaurant]_A in_S [the park]_A.            - cf. guidelines "There are earrings on the table"
> There_F is [a restaurant]_A nearby_S+A.
> There_F is [a great restaurant]_A in_S [the park]_A.
> There_F is [a great restaurant]_A hidden_S [in the park]_A.   - cf. guidelines "There are lots of them lurking in the bushes"
> There_F is a wedding_P.
> There_F is a wedding_P [in the park]_A.
>
> No need for parallel scenes with this analysis!
